components/Profile/ConnectFurther.js

```diff
@@ -13,10 +13,10 @@ import Instagram from '../../assets/jsicons/socialmedia/Instagram';
 import IgAdd from '../../assets/jsicons/socialmedia/IgAdd';
 import Facebook from '../../assets/jsicons/socialmedia/Facebook';
 import FbAdd from '../../assets/jsicons/socialmedia/FbAdd';
-import LinkedIn from '../../assets/jsicons/socialmedia/LinkedIn';
-import LinkedInAdd from '../../assets/jsicons/socialmedia/LinkedInAdd';
-import GitHub from '../../assets/jsicons/socialmedia/GitHub';
-import GitHubAdd from '../../assets/jsicons/socialmedia/GitHubAdd';
+// import LinkedIn from '../../assets/jsicons/socialmedia/LinkedIn';
```

Comment:
Why did we comment this out instead of just removing it?

Reply:
Ari put those in, but they were throwing errors so I commented them out. they will be used as soon as we create columns for them in the backend